lib/core/options/options.js

```diff
@@ -80,3 +81,47 @@ export function set(opaqueId: OpaqueModuleId, optionKey: string, value: mixed) {
 
 	return true;
 }
+
+function getModified() {
+	return filterMap(Modules.all(), module => {
+		const { moduleID, options } = module;
+
+		const enabledByDefault = !module.disabledByDefault;
+		const isEnabled = Modules.isEnabled(module);
+		const moduleStatusChanged = enabledByDefault !== isEnabled;
+
+		const modifiedOptions = isEnabled ? _.omitBy(options, v => _.isEqual(v.value, v.default)) : {};
+
+		if (!_.isEmpty(modifiedOptions) || moduleStatusChanged) {
+			return [{
+				moduleID,
+				modifiedOptions,
+				moduleStatus: moduleStatusChanged ? `${enabledByDefault ? 'on' : 'off'} → ${isEnabled ? 'on' : 'off'}` : '',
+			}];
+		}
+	});
+}
+
+export function getModifiedText(types: Array<*> = ['text', 'boolean', 'enum']) {
+	const lines = [];
+
+	let optionMaxLength = 0;
+	let moduleMaxLength = 0;
+	for (const { moduleID, modifiedOptions, moduleStatus } of getModified()) {
+		moduleMaxLength = Math.max(moduleMaxLength, moduleID.length);
+		optionMaxLength = Math.max(optionMaxLength, ...Object.keys(modifiedOptions).map(v => v.length));
+
+		lines.push(() =>
+			[
+				`    ${_.padEnd(moduleID, moduleMaxLength)} ${moduleStatus}`,
```

Comment:
Should be able to use native `.padEnd` http://kangax.github.io/compat-table/es2016plus/#test-String_padding